spec/lib/date_time_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'date'
+require 'date_time'
+
+describe DateTime do
+    it 'get correct beginning of rice fiscal year' do
+        allow(DateTime).to receive(:now).and_return(Date.parse("2019-09-28"))
+        year1 = DateTime.beginning_of_rice_fiscal_year
```

Comment:
maybe not `year1` for var name b/c it isn't a year value.  could just say

`expect(DateTime.beginning_of_rice_fiscal_year).to eq DateTime.parse("2019-07-01")` 

on this line and similar expectations after each `allow` line

Reply:
Done